Formula/scw.rb

```diff
@@ -1,35 +1,29 @@
 class Scw < Formula
-  desc "Manage BareMetal Servers from command-line (as easily as with Docker)"
+  desc "Command-line Interface for Scaleway"
   homepage "https://github.com/scaleway/scaleway-cli"
-  url "https://github.com/scaleway/scaleway-cli/archive/v1.20.tar.gz"
-  sha256 "4c50725be7bebdab17b8ef77acd230525e773631fef4051979f8ff91c86bebf8"
-  head "https://github.com/scaleway/scaleway-cli.git"
-
-  bottle do
```

Comment:
Please don't change the bottle block

Reply:
I did it to fix a test asking be to not have a bottle block: https://github.com/Homebrew/homebrew-core/runs/877136822#step:6:39

Reply:
That was on the other formula

Reply:
I've putted back the bottle block from the current version in scw@1

Reply:
@SMillerDev It fails if I put the bottle block back: https://github.com/Homebrew/homebrew-core/pull/57158/checks?check_run_id=878051256#step:6:39 Are you sure that I don't have to take it out because scw@1 is a new formula?

Reply:
It needs to _only_ be in scw.rb

Reply:
The problem is that we don't have those values for the new version. How could we get the right value for the scw.rb? We only got the value for the scw@1 at the moment.

Reply:
It needs to be the value that's currently in the file. As far as brew is concerned you are doing an update to the scw formula and creating a new formula. New formula can't get bottle values (because we don't expect people to have all macOS versions) and existing formulae shouldn't have their bottle block deleted on updates.